certification/formatters/generic.go

```diff
@@ -39,8 +37,7 @@ func genericXMLFormatter(ctx context.Context, r runtime.Results) ([]byte, error)
 
 	responseJSON, err := xmlMarshalIndent(response, "", "    ")
 	if err != nil {
-		e := fmt.Errorf("%w with formatter %s: %s",
-			errors.ErrFormattingResults,
+		e := fmt.Errorf("error formatting results with formatter %s: %w",
 			"json",
```

Comment:
Should this be `xml`?

Reply:
also `responseXML?`

Reply:
That looks like that's been there a while. I'll fix though.

Reply:
Thanks for cleaning this up too!